pkg/agent/cniserver/pod_configuration_linux.go

```diff
@@ -113,3 +113,9 @@ func (pc *podConfigurator) reconcileMissingPods(ifConfigs []*interfacestore.Inte
 		klog.Warningf("Interface for Pod %s/%s not found in the interface store", ifaceConfig.PodNamespace, ifaceConfig.PodName)
 	}
 }
+
+func (pc *podConfigurator) init() {
+
+}
+
+func (pc *podConfigurator) Run(stopCh <-chan struct{}) {}
```

Comment:
It should probably pending on stopCh to make the behaviors consistent (not returning until stopCh is closed)

Reply:
Unify the behaviors to make caller platform agnostic. Otherwise callers will see the function to be returned on channel stop on windows and to be returned immediately on linux.